Resources/Textures/Interface/Inventory/meta.json

```diff
@@ -0,0 +1,182 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-NC-SA-3.0",
+    "copyright": "Taken from https://github.com/goonstation/goonstation at commit e77d85d9c1d93aa32da7702737ceeac2b56738ac.",
+    "states": [
+        {
+            "name": "back",
+            "directions": 1,
+            "delays": [
+                [
+                    1.0
+                ]
+            ]
```

Comment:
Don't need these if it's default anymore

Reply:
https://github.com/space-wizards/RSI-editor/issues/8